workflow/resources/config/default.yaml

```
@@ -231,6 +233,21 @@ calling:
         labels:
           clinical significance: pathogenic, risk factor, or drug response
         subcategory: known somatic variants
+      
+      somatic_splice_mutation:
```

```suggestion
      splice_variants:
```
just splice variants, in the other callsets we also do not explicitly mention somatic in the name although it is.